requirements.txt

```diff
@@ -5,7 +5,6 @@ pytest-cov==2.6.0
 wheel==0.32.1
 paho-mqtt==1.4.0
 transitions==0.6.8
-enum34==1.1.6
 flake8
```

Comment:
just double-checking that this is intentional.

Reply:
yup

---
In reply to: [240414068](https://github.com/Azure/azure-iot-sdk-python-preview/pull/34#discussion_r240414068) [](ancestors = 240414068)